src/api/applicant/content-types/applicant/lifecycles.js

```diff
@@ -1,14 +1,76 @@
+const lodash = require('lodash');
+const fetch = require('node-fetch');
 const { createModuleResolutionCache } = require("typescript");
+const { Webhook, MessageBuilder } = require('discord-webhook-node');
+
+const frontend_url_sanitized = process.env.FRONTEND_URL.replace(/["']/g, "");
+
+// use markdown block code to format this as JSON
+const formatJSON = (jsonData) => {
+  return "```JSON\n" + JSON.stringify(jsonData, null, 1) + "\n```";
```

Comment:
Consider using an npm package like `@braintree/sanitize-url` and use its more comprehensive `sanitizeUrl` method

Reply:
Actually this is not to sanitize URL but to format a string as JSON markdown. Do you have suggestions to better implement this?